tests/fixtures_utils.py

```diff
@@ -51,16 +52,16 @@ def _load_file_tsv(file_path: str) -> Tuple[List[str], List[str]]:
     return time, v
 
 
-def _load_file_h5(file_path: str) -> Tuple[List[str], List[str]]:
+def _load_file_h5(
+    file_path: str, sampling_rate_construct: int, x_range: Optional[Tuple[int, int]]
+) -> Tuple[List[str], List[str]]:
     wf = WellFile(file_path)
     tissue_data = wf.get_raw_tissue_reading()
-    # only analyze first 21 seconds of data so png graphs are readablw
-    last_index = 0
-    for idx, t in enumerate(tissue_data[0]):
-        last_index = idx
-        if t > CENTIMILLISECONDS_PER_SECOND * 21:
-            break
-    return tissue_data[0][:last_index], tissue_data[1][:last_index]
+    if x_range is not None:
```

Comment:
minor style point: fewer indented lines of code are better in general (if it doesn't affect flow, readability, etc.), so this could be `if x_range is None: return tissue_data[0], tissue_data[1]`, and then you'd only have 1 indented line instead of 3.   Tracking nested levels of indentation makes the code slightly harder to read, and you also have less space until you run into the ~88 character limit before lines start to be text-wrapped